tests/colorChange.phpt

```diff
@@ -8,7 +8,7 @@ use Tester\Assert;
 
 $expected = array(
     "336699" => "264d73",
-    "913399" => "6d2673"
+    "913399" => "6c2673"
```

Comment:
I think these test cases should not be changed after fixing the deprecated warning message on PHP 8.1 version.

If test cases has been changed after modifying some codes, it means modified and original code behaviors are different.

It will not be compatible with previous released version.